projects/ui/src/lib/services/po-notification/po-notification.service.spec.ts

```diff
@@ -99,5 +103,36 @@ describe('PoNotificationService:', () => {
       notificationService['observableOnClose'](fakeRef);
       expect(spy).toHaveBeenCalled();
     });
+
+    it('must destroy toaster when `stackTop` is more than 5', fakeAsync(() => {
```

Comment:
Usamos should como padrão.

```suggestion
    it('should destroy toaster when `stackTop` is more than 5', fakeAsync(() => {
```